CareKit/CareKit/iOS/Task/SliderTaskView.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  SliderTaskView.swift
+//
+//  Created by Dylan Li on 5/26/20.
+//  Copyright © 2020 NetReconLab. All rights reserved.
+//
+#if !os(watchOS)
+
+import CareKitStore
+import CareKitUI
+import Foundation
+import SwiftUI
+
+/// A card that updates when a controller changes. The view displays a header view, multi-line label, and a completion button.
+///
+/// In CareKit, this view is intended to display a particular event for a task. The state of the button indicates the completion state of the event.
+///
+/// # View Updates
+/// The view updates with the observed controller. By default, data from the controller is mapped to the view. The mapping can be customized by
+/// providing a closure that returns a view. The closure is called whenever the controller changes.
+///
+/// # Style
+/// The card supports styling using `careKitStyle(_:)`.
+///
+/// ```
+///     +-------------------------------------------------------+
```

Comment:
Was not sure how to generate a visual representation for the slider card. If you point me towards how to generate it I can do it.